src/lib/components/shared/ChevronCard.svelte

```diff
@@ -6,10 +6,10 @@
 	export let chevronData;
 </script>
 
-<li>
+<li class="chevron-card">
 	<div>
 		<h3>
-			<a href={chevronData.web_url} class="chevron-card">{chevronData.heading}</a>
+			<a class="chevron-card__link link-effects link--medium link--bold" href={chevronData.web_url}>{chevronData.heading}</a>
```

Comment:
I'm pretty sure you could nest this, so to avoid having to create the `chevron-card__link` class you could nest the stye so it applies to all `a` tags within `chevron-card`s. I'm not sure how idiomatic it is, but might be worth it if you never plan to have a link in a chevron card that you want to style differently

Reply:
Never tried CSS nesting before, but I think it would clash with BEM as you wouldn't be able to work out what the class is doing just from it's name: `chevron-card`. By nesting, do you mean coding it like below? (I think native CSS nesting has only been out for 6 months or so, interesting...)

```
.chevron-card {

  a:before {
    /* some styles */
  }
  
    a:after {
    /* some styles */
  }

  /* other chevron-card only styles */

}

```

Honestly, not sure which approach is better. Nesting reduces clutter of class names in markup and better organises/scopes the CSS in general, but deviates from BEM convention. Keeping `chevron-card__link` follows BEM as it's more semantic but bloats markup.